site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassification.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A
+ *
+ * @ORM\Table(name="ProjectClassification")
+ * @ORM\Entity
+ */
+class ProjectClassification
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="categories", type="text")
+     */
+    private $categories;
+
+    /**
+     *
+     *
+     * @var Project
+     *
+     * @ORM\ManyToOne(targetEntity="Project", inversedBy = "projects")
+     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
```

Comment:
SET NULL is not the right method here. If deleting a Project the ProjectClassification entities should also be deleted. If deleting a ProjectClassification nothing should happen to the Project.